src/runrms/executor/_rms_executor.py

```diff
@@ -101,6 +39,20 @@ def _config_env(self) -> dict[str, str]:
         config_env.update(version_env)
         return config_env
 
+    @property
+    def config(self) -> InteractiveRMSConfig | FMRMSConfig:
+        return self._config
+
+    def update_exec_env(self, key: str, val: str) -> None:
```

Comment:
This is only called two times:
- Once in the interactive executor to update the QT scale factor from the CLI
- Once in the forward model executor to update the batch license, which is optionally set from the configuration

Hence with Ert's exec_env.json now gone we can make this dramatically simpler. The `*PATH` vars are just there as a safely mechanism but are not actually used